### ISSUE TITLE
Change the title

### ISSUE THREAD

**Issue by jfoclpf** [jfoclpf/autocosts#112]
According to SEO optimization, the title should have 50 to 60 characters

Therefore we should make on server side, before assigning unexistant strings to UK's trings

```
seo_title = web_page_title + (sub_title2 ? " - "  + sub_title2 : "")`;
```

**Comment by jfoclpf** [comment]
Title

 * Length: Title tags should be 50-60 characters long, including spaces.
 * Keyword placement: Your most important keywords need to be first in your title tag, with your least important words coming last.
 * Brand name: If your company name is not part of the important keyword phrases, put it at the end of the title tag.
 * Do not duplicate title tags: They must be written differently for every page. Don’t mass replicate your title tags it will negatively affect your search visibility.
 * Make it relevant: Title tags must accurately describe the content on the page.
 * Do not ‘keyword stuff’ title tags: these are badly written title tags that try to rank for everything or repeat a word over and over. Keyword stuffing is the worst offense when it comes to title tags and you will be penalised for it.
 * Make your headline (<h1> tag) different from the title tag: This is another opportunity to vary the keyword phrasing of your page and increase its chances of appearing for different search intent.

in: https://searchenginewatch.com/2016/05/16/how-to-write-meta-title-tags-for-seo-with-good-and-bad-examples/#checklist

```js
seo_title = web_page_title + (sub_title2 ? " - "  + sub_title2 : "")`;
```

**Comment by jfoclpf** [comment]
Solved with 72a0d18e4b06cfd60f561fefcbae39fdb9d296f0
